public/app/features/alerting/unified/mocks/server/handlers.ts

```diff
@@ -0,0 +1,10 @@
+/**
+ * Contains definitions for all handlers that are required for test rendering of components within Alerting
+ */
+
+import { HttpResponse, http } from 'msw';
+
+import { defaultAlertmanagerChoiceResponse } from 'app/features/alerting/unified/mocks/alertmanagerApi';
+
+export const alertmanagerChoiceHandler = (response = defaultAlertmanagerChoiceResponse) =>
```

Comment:
I'd split these into an object that maps to our RTKQ APIs like `alertRuleApi` `alertmanagerApi`

Reply:
I was trying to avoid tying the API and our consumption/implementation together too closely, as that means that refactoring the RTKQ code would necessitate refactoring our test code more significantly. Are you okay if we leave the two separate?